rollup.config.js

```diff
@@ -18,35 +18,35 @@ function external(id) {
 const cjs = [
   {
     input,
-    output: { file: `cjs/${pkg.name}.js`, format: 'cjs' },
+    output: { file: `cjs/${pkg.name}.js`, format: 'cjs', exports: 'named' },
     external,
     plugins: [
-      babel({ exclude: /node_modules/ }),
+      babel({ exclude: /node_modules/, babelHelpers: 'bundled' }),
```

Comment:
The new version of the babel plugin uses a different setting for whether to include the babel runtime, so this changed throughout the fiel.